arch/arm64/boot/dts/qcom/sdm450-motorola-ali.dts

```diff
@@ -121,10 +167,13 @@
 		reg = <0x38>;
 		interrupt-parent = <&tlmm>;
 		interrupts = <65 IRQ_TYPE_EDGE_FALLING>;
+
 		vcc-supply = <&pm8953_l10>;
+		iovcc-supply = <&pm8953_l6>;
 
-		pinctrl-names = "default";
-		pinctrl-0 = <&ts_int_active &ts_reset_active>;
+		pinctrl-names = "default", "sleep";
```

Comment:
pinctrl-0
pinctrl-1
pinctrl-names
If driver does not implement default, sleep for pinctrls then you should only use default.
So original solution was fine.